toolz/compatibility.py

```diff
@@ -1,34 +1,6 @@
-import operator
-import sys
-PY3 = sys.version_info[0] > 2
-PY34 = sys.version_info[0] == 3 and sys.version_info[1] == 4
-PYPY = hasattr(sys, 'pypy_version_info')
+__all__ = ('PYPY', 'PY3')
 
-__all__ = ('map', 'filter', 'range', 'zip', 'reduce', 'zip_longest',
-           'iteritems', 'iterkeys', 'itervalues', 'filterfalse',
-           'PY3', 'PY34', 'PYPY')
+import sys
 
-if PY3:
```

Comment:
Can we keep all of these definitions instead of deleting them?  Other packages may refer to `toolz.compatibility`.  Note that we don't need to use these in `toolz`, so no need to revert other modules that you've changed.

Reply:
Good point.  I'll restore them. Maybe a deprecation warning should be added to the module.